perftest_qos_profiles.xml

```diff
@@ -250,8 +250,8 @@ _KeepDurationUsec variable name corresponds to the command-line option -keepDura
 
       </participant_qos>
 
-      <!-- === Publisher Configuration: ================================= 
-        
+      <!-- === Publisher Configuration: =================================
```

Comment:
Revert this file since we are not touching it.

Reply:
Done